client/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { h, render } from "preact";
+
+import Reporter from "./components/Reporter";
+
+import "./style/index.css";
+
+window.addEventListener("load", () => {
```

Comment:
The only problem here is, how do I get the instance of `preact` render function in the `ejs` file.

In this case, everything will be bundle during `webpack` and I don't have to care about resolving the modules.